src/main/java/com/java/dojo/recipes/entities/Recipe.java

```diff
@@ -0,0 +1,30 @@
+package com.java.dojo.recipes.entities;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Getter
+@Setter
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+@Builder
+public class Recipe {
+	
+	@Id @GeneratedValue(strategy = GenerationType.IDENTITY)
+	private Long id;
+	private List<String> ingredients = new ArrayList<>();
+	private List<String> preparingMode = new ArrayList<>();
+	
```

Comment:
- quando a gente mapeia entidades, você pode omitir a inicialização destes objetos, ou seja, remover `= new ArrayList<>()`
- se você estiver rodando no IntelliJ você deve estar vendo um warning como esse https://stackoverflow.com/questions/21059451/cannot-declare-list-property-in-the-jpa-entity-class-it-says-basic-attribute
  - a gente precisa criar uma classe e representar isso com um relacionamento 1 -> N 
  - algo como `@OneToMany private List<Ingredient> ingredients;`